### PR TITLE
Remove authentication from GraphiQL and the session store.

### PR DESCRIPTION
**Pull request by alloy**
Which was an in-memory session store. Let’s see what this does for our memory graph.

In terms of security, this didn’t really offer any. Actual GraphQL requests could still be made, so nefarious people could still do their thing, whereas we legitimate users had to jump through some hoops (and eg. could not share query links).

### CODE REVIEW

**Comment by orta** [comment]
I rubber ducked this with @dblock - I think this is fine.

File: index.js [hunk 1]
```diff
@@ -25,21 +23,14 @@ const { PORT, NODE_ENV, GRAVITY_API_URL, GRAVITY_ID, GRAVITY_SECRET } = process.
 
 const app = express()
 const port = PORT || 3000
-const sess = {
-  secret: GRAVITY_SECRET,
-  cookie: {},
-}
```
Comment (joeyAghion) [inline index.js]: Was MP using its API secret (w/Gravity) to encrypt its own session? I don't think that's necessary or advised. 😬

**Comment by joeyAghion** [comment]
+1 to dropping sessions.

I had a small trip down memory lane revisiting my old worries (https://github.com/artsy/metaphysics/issues/333#issuecomment-281802826) about providing the public a very convenient UI for lighting our servers on fire. I'm still slightly worried about this but am OK with this change. Please keep an eye out for bad actors, and we should explore how alerting might help with this.

P.S. Are there config values that we can now unset?

File: index.js (hunk 1 above)
Comment (orta) [reply index.js]: Looks like it, yeah, I _think_ we can remove that client app. 

Not sure whether it's used anywhere in logging though.

**Comment by orta** [comment]
Did a quick audit, only `BASIC_AUTH_PASSWORD` and `BASIC_AUTH_USERNAME`

**Comment by alloy** [comment]
> `OPTICS_API_KEY`,  `BASIC_AUTH_PASSWORD` and `BASIC_AUTH_USERNAME`

These three can definitely go.

> `MEMCACHIER_PAS`, `MEMCACHIER_SERVERS` and `MEMCACHIER_USERNAME`

These three definitely not.

> `NODE_MODULES_CACHE`

This probably needs testing

**Comment by orta** [comment]
the memcachiers aren't referenced anywhere in the source FWIW

**Comment by alloy** [comment]
Interesting. We do use memcachier. Maybe these are automatically set by the Heroku addon, we just don’t use them?

**Pull request merged**